### ISSUE TITLE
wrong SDP param

### ISSUE THREAD

**Issue by pkeroulas** [garethsb/nmos-cpp-examples#2]
Hi Gareth,

Thanks for this example,that I could easily compile. However, I can't make it work when patching (IS-05) a ST2110 video or a simple audio:
```
 Transport file error - sdp processing error - unsupported transport or format-specific parameters
```
It is hard to get more detail, as the nmos-cpp is pre-built. While I'm trying to modify the lib for more verbosity, please can you take a look at the following SDP to spot any obviously wrong params? 

Audio SDP:
```
o=- 1443716955 1443716955 IN IP4 <my-ip>
s=<my audio sender>
t=0 0
a=group:DUP primary secondary
m=audio 20000 RTP/AVP 97
c=IN IP4 239.164.9.61/64
a=source-filter: incl IN IP4 239.164.9.61  <source IP>
a=rtpmap:97 L24/48000/6
a=mediaclk:direct=0
a=framecount:6
a=ptime:0.125
a=ts-refclk:ptp=IEEE1588-2008::.......................................
a=mid:primary
m=audio 20000 RTP/AVP 97
c=IN IP4 239.164.137.151/64
a=source-filter: incl IN IP4 239.164.137.151 <source-IP>
a=rtpmap:97 L24/48000/6
a=mediaclk:direct=0
a=framecount:6
a=ptime:0.125
a=ts-refclk:ptp=IEEE1588-2008:.......................................
a=mid:secondary
```

Thanks

**Comment by garethsb** [comment]
And I assume the actual SDP data does have a `v=0` line at the beginning!? I think you'd get a different parse error in that case, so I'm sure you do...

**Comment by pkeroulas** [comment]
Good catch! I have just noticed `channel_count` in the json config. It works for audio.
The error for video patch must be very similar.
I'll find a way to tweak my node_implementation to match our audio mapping (audio1 is 5.1, audio2 is stereo .... )

**Comment by garethsb** [comment]
Yes, one of the limitations of nmos-cpp-node is that all Senders and Receivers of each type have the same format. But a small code change would enable you to have a different format for each one.
